CONTRIBUTION.md

```diff
@@ -0,0 +1,235 @@
+
+## 3. Creating a Pull Request (PR)
+
+Once you've implemented your changes, the next step is to submit them for review by creating a 
+Pull Request (PR). A PR is a formal request to merge your changes from your forked repository into 
+the main project. This section outlines the process for creating a well-structured PR that 
+facilitates efficient review and integration of your contributions. By following these guidelines, 
+you ensure your code is presented clearly and is ready for consideration.
+
+### Step-by-Step Guide
+
+#### 1. Create a New Branch
+   Always create a new branch to work on your changes. Avoid making changes directly on the `develop` or `master` branch.
+
+   ```bash
+   git checkout -b feature/my-new-feature
+   ```
+   
+#### 2. Make Your Changes
+Implement the required changes or new features. Make sure to write clean, well-commented, and readable code. If applicable, include tests and documentation for the new functionality.
+
+#### 3. Commit Your Changes
+Once you’ve made your changes, commit them with a clear and descriptive message. The commit message should follow this structure:
+
+```
+[TYPE] Short description of the changes
+```
+Types:
+
+* `feat`: A new feature
+* `fix`: A bug fix
+* `docs`: Documentation changes
+* `refactor`: Code refactoring (no feature change)
+* `test`: Adding or modifying tests
+
+Example:
+
+   ```bash
+   git commit -m "feat: add login functionality"
+   ```
+
+#### 4. Push Your Changes
+After committing your changes, push them to your fork:
+
+   ```bash
+    git push origin feature/my-new-feature
+   ```
+
+#### 5. Create a Pull Request
+
+1. Go to your fork on GitHub.
+
+2. Click on the "New Pull Request" button.
+
+3. Select the base repository and base branch (usually `develop`), then select the branch you just pushed.
+
+4. Fill Out the PR Template
+
+   Make sure to fill out the PR template provided. The template helps us better understand your change. Typically, an PR will require the following information:
+
+   * Add a title and description for the PR. The description should include:
+     * What was changed and why.
+     * Any related issues.
+     * Any additional context if necessary (e.g., relevant screenshots or breaking changes). 
+   
+   Once everything looks good, submit the PR for review.
+
+### PR Review and Feedback
+
+Once the PR is submitted, the team will review it. Be prepared to:
+
+* Address any feedback or requested changes.
+* Keep your branch up to date with the base branch by rebasing or merging.
+
+## 4. Additional Notes
+
+* **Testing:** Please ensure that your code is well-tested. If your changes introduce new features or bug fixes, add appropriate tests to verify the behavior.
+
+* **Documentation**: Update relevant documentation to reflect any new features or changes to existing functionality.
```

Comment:
What do we mean by "Documentation" here? Our product docs aren't currently open source.
Do we mean code comments perhaps?